src/wlcsim/params.f03

```diff
@@ -992,19 +1030,23 @@ subroutine initialize_wlcsim_data(wlc_d, wlc_p)
 #if MPI_VERSION
         call init_MPI(wlc_d)
 #endif
-        allocate(wlc_d%R(NT,3))
-        allocate(wlc_d%U(NT,3))
+        allocate(wlc_d%R(3,NT))
+        allocate(wlc_d%U(3,NT))
         if (wlc_p%codeName /= 'bruno' .OR. wlc_p%nInitMCSteps /= 0) then
-            allocate(wlc_d%RP(NT,3))
```

Comment:
these can be pulled out of the if statement now actually, can we do that here? thanks!